test/sql/test_time_fn/T/test_time_fn

```diff
@@ -181,3 +181,9 @@ select d3, dow_2, previous_day(d3, dow_2) from previous_dow order by d3;
 select d3, dow_3, previous_day(d3, dow_3) from previous_dow order by d3;
 select d3, dow_full, previous_day(d3, dow_full) from previous_dow order by d3;
 select d1, dow_2, previous_day(d1, dow_2) from previous_day_test a join previous_dow b on a.d1 = b.d3 order by d1;
+
+
+-- name: test_week_iso
+select week_iso("2023-01-01");
+select week_iso("2023-01-02");
+select week_iso("2023-01-03");
```

Comment:
can you add a case to test `week_iso(null)`

Reply:
Do you mean week_iso("")？

Reply:
> can you add a case to test `week_iso(null)`
week_iso(null) and week_iso("") has added to the test-sql.